Source/WebCore/platform/graphics/cg/GraphicsContextCG.cpp

```diff
@@ -167,6 +168,22 @@ static void setCGBlendMode(CGContextRef context, CompositeOperator op, BlendMode
     CGContextSetBlendMode(context, selectCGBlendMode(op, blendMode));
 }
 
+static void setCGContextPath(CGContextRef context, const Path& path)
+{
+    CGContextBeginPath(context);
+    addToCGContextPath(context, path);
+}
+
+static void drawPathWithCGContext(CGContextRef context, CGPathDrawingMode drawingMode, const Path& path)
```

Comment:
I think adding `WithCGContext` is redundant and uncommon since it can be deduced from the first argument.

Reply:
The name resolution doesn't work this way. GraphicsContextCG::drawPath conflicts with this.
I'll land with this name.
I'll remove the function altogether when removing the ifdef.